docs/sentiment-analysis.md

```diff
@@ -1,2 +1,53 @@
 # Example 4: Run sentiment analysis using HuggingFace and store results in your data warehouse
 
+Sentiment Analysis is a powerful technique that takes advantage of NLP to identify affective states. It is usually used to identify customer satisfaction. For simplicity we are taking advantage of a pretrained model and going to use [HuggingFace transformers for sentiment analysis](https://huggingface.co/transformers/quicktour.html).
```

Comment:
"Sentiment Analysis is a powerful technique that takes advantage of NLP to identify affective states." This sounds way too technical

Reply:
ok ok :D